docker/auto-setup.sh

```diff
@@ -281,34 +281,40 @@ setup_es_index() {
 
 register_default_namespace() {
     echo "Registering default namespace: ${DEFAULT_NAMESPACE}."
-    if ! tctl --ns "${DEFAULT_NAMESPACE}" namespace describe; then
+    if ! temporal operator namespace describe "${DEFAULT_NAMESPACE}"; then
```

Comment:
Is temporal cli already available here? what is the version of that cli?

Reply:
Yes, it is. It's a submodule, but I don't think it's being auto updated. (cc: @mindaugasrukas)
I'm updating it in this commit to master.